src/Sylius/Behat/Context/Setup/ShippingContext.php

```diff
@@ -546,6 +546,18 @@ public function thisShippingMethodIsOnlyAvailableForOrdersWithATotalWeightLessOr
         $this->addRuleToShippingMethod($rule, $shippingMethod);
     }
 
+    /**
+     * @Given /^(this shipping method) has been disabled$/
+     */
+    public function thisShippingMethodHasBeenDisabled(ShippingMethodInterface $shippingMethod): void
+    {
+        /** @var ShippingMethodInterface $shippingMethod */
+        $shippingMethod = $this->shippingMethodRepository->findOneBy(['code' => $shippingMethod->getCode()]);
```

Comment:
Why is this necessary?

Reply:
Yup, for some reason, the ShippingMethod object is not tracked by the entity manager 🤷🏼‍♂️.